src/metabase/models/model_index.clj

```diff
@@ -143,7 +143,8 @@
               (doseq [deletions-part (partition-all 10000 deletions)]
                 (t2/delete! ModelIndexValue
                             :model_index_id (:id model-index)
-                            :model_pk [:in (->> deletions-part (map first))])))
+                            :model_pk [:in (->> deletions-part (map first))])
+                (search/delete! #_ModelIndexValue "indexed-entity" (->> deletions-part (map first)))))
```

Comment:
nit pick, but i think pulling out this shared expression into a `model-index-ids` let is worth it for the clarity and saved repetition.